mitiq/calibration/settings.py

```diff
@@ -336,6 +338,22 @@ def make_problems(self) -> List[BenchmarkProblem]:
             elif circuit_type == "rb":
                 circuit = generate_rb_circuits(num_qubits, depth)[0]
                 ideal = {"0" * num_qubits: 1.0}
+            elif circuit_type == "rotated_rb":
+                theta = benchmark["theta"]
+                if num_qubits == 1:
+                    circuit = generate_rotated_rb_circuits(num_qubits, depth)[
+                        0
+                    ]
+                    ideal = {
+                        "0": (2 / 3) * np.sin(theta / 2) ** 2,
```

Comment:
Why not using a variable, say `p`, for this expression and then `1 - p` as expression for the "1" expression.

Reply:
good point, that is cleaner